pyvista/core/grid.py

```diff
@@ -264,11 +265,18 @@ def meshgrid(self) -> list[NumpyArray[float]]:
 
         Returns
         -------
-        list[numpy.ndarray]
-            List of numpy arrays representing the points of this mesh.
+        tuple[numpy.ndarray]
+            Tuple of numpy arrays representing the points of this mesh.
```

Comment:
Should we mention the length 3 either on the type line or the short description?

Reply:
Put in length three explicitly into the tuple here.